h2o-core/src/main/java/water/rapids/Merge.java

```diff
@@ -13,6 +13,9 @@
 import static water.rapids.SingleThreadRadixOrder.getSortedOXHeaderKey;
 
 public class Merge {
+  
+  public static int ASCENDING = 1;
```

Comment:
Are these constant used somewhere else?

Reply:
No, I introduced them to and the code coming with this PR is the first user. I wanted to get rid of magical constants in the code. I myself misunderstood this once and used 0 for descending order. :rabbit: